controllers/user.js

```diff
@@ -1246,6 +1248,9 @@ async function postRemovalKan(req, res) {
     });
   }
   await DB.removeKan(sessionUser);
+  if (req.session?.kanary) {
+    req.session.kanary.onRemovalPilotList = false;
```

Comment:
Same Q as above. I believe this is checking that `req.session` is non-null before checking the `.kanary` property, but do we similarly need to check if `req.session.kanary` is non-null before setting `.onRemovalPilotList`?